exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/xsort/managed/SortConfig.java

```diff
@@ -114,15 +127,25 @@ public SortConfig(DrillConfig config) {
     } else {
       mSortBatchSize = Character.MAX_VALUE;
     }
+
+    // Disable the single-batch shortcut; primarily for testing
+
+    if (config.hasPath(ExecConstants.EXTERNAL_SORT_OPTIMIZE_SINGLE_BATCH)) {
+      enableSingleBatchShortcut = config.getBoolean(ExecConstants.EXTERNAL_SORT_OPTIMIZE_SINGLE_BATCH);
+    } else {
```

Comment:
Minor comment - can save 3 LOC by initializing to "true" (and dropping the "final") , then no need for the else { }